subsys/mgmt/mcumgr/util/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2018-2021 mcumgr authors
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# MCUmgr utilities, for use within the library.
+# API interface for utilities is exposed by mgmt_mcumgr_util,
+# and should not be exposed outside of mgmt_mcumgr.
+add_library(mgmt_mcumgr_util src/zcbor_bulk.c)
```

Comment:
Need a task to move zcbor_bulk over to zcbor itself and remove the implementation from mcumgr

Reply:
Working on it.